src/index.js

```diff
@@ -28,21 +28,21 @@ export function rangeFromLineNumber(textEditor: TextEditor, line: ?number, colum
   }
 
   const columnGiven = typeof column === 'number' && Number.isFinite(column) && column > -1
-  let lineText = buffer.lineForRow(lineNumber)
-  const lineLength = lineText.length
-  if (columnGiven && column > lineLength) {
-    throw new Error(`Column start (${column}) greater than line length (${lineLength})`)
-  } else if (columnGiven) {
-    lineText = lineText.substr(column)
-  }
-
-  let colEnd = lineLength
+  const lineText = buffer.lineForRow(lineNumber)
+  let colEnd = lineText.length
   let colStart = columnGiven ? column : 0
-  const match = Helpers.getWordRegexp(textEditor, [lineNumber, colStart]).exec(lineText)
-  if (match) {
-    colEnd = colStart + match.index + match[0].length
-    if (!columnGiven) {
-      colStart = match.index
+  if (columnGiven && column > lineText.length) {
```

Comment:
Any reason this check is behind the `columnGiven` flag? Shouldn't it be at the end of all figuring out of the end column?